<ui version="4.0" >
 <class>RegPanelBase</class>
 <widget class="QDialog" name="RegPanelBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Register Test Panel</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="topLabel" >
         <property name="maximumSize" >
          <size>
           <width>32767</width>
           <height>40</height>
          </size>
         </property>
         <property name="text" >
          <string>Register test info for module ...</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer4" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="textLabel4" >
         <property name="text" >
          <string>Chip</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="chipBox" >
         <property name="maximum" >
          <number>15</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Q3GroupBox" name="globalBox" >
       <property name="title" >
        <string>Global Register Test</string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="statusGRLabel" >
              <property name="text" >
               <string>Test result</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer1" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>61</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="statusGR" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="text" >
               <string>?</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer2" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>61</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLineEdit" name="bitsGR" />
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Q3GroupBox" name="pixelBox" >
       <property name="title" >
        <string>Pixel Register test</string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="statusCP0Label" >
              <property name="text" >
               <string>Test result CP0</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer1_2" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>21</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="statusCP0" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="autoFillBackground" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>?</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer2_2" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>22</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="bitsCP0" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="statusCP1Label" >
              <property name="text" >
               <string>Test result CP1</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer1_2_2" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>21</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="statusCP1" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="text" >
               <string>?</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer2_2_2" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>22</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="bitsCP1" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="statusCP2Label" >
              <property name="text" >
               <string>Test result CP2</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer1_2_3" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>21</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="statusCP2" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="text" >
               <string>?</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer2_2_3" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>22</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="bitsCP2" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="statusCP3Label" >
              <property name="text" >
               <string>Test result CP3</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer1_2_4" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>21</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="statusCP3" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="text" >
               <string>?</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer2_2_4" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>22</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="bitsCP3" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="statusCP4Label" >
              <property name="text" >
               <string>Test result CP4</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer1_2_4_4" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>21</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="statusCP4" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="text" >
               <string>?</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer2_2_4_4" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>22</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="bitsCP4" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="statusCP5Label" >
              <property name="text" >
               <string>Test result CP5</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer1_2_4_5" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>21</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="statusCP5" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="text" >
               <string>?</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer2_2_4_5" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>22</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="bitsCP5" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="statusCP6Label" >
              <property name="text" >
               <string>Test result CP6</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer1_2_4_6" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>21</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="statusCP6" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="text" >
               <string>?</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer2_2_4_6" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>22</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="bitsCP6" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="statusCP7Label" >
              <property name="text" >
               <string>Test result CP7</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer1_2_4_7" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>21</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="statusCP7" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="text" >
               <string>?</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer2_2_4_7" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>22</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="bitsCP7" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="statusCP8Label" >
              <property name="text" >
               <string>Test result CP8</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer1_2_4_8" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>21</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="statusCP8" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="text" >
               <string>?</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer2_2_4_8" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>22</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="bitsCP8" />
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <spacer name="spacer3" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>341</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton" >
         <property name="text" >
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
